br/pkg/lightning/importer/import.go

```diff
@@ -294,6 +295,10 @@ func NewImportControllerWithPauser(
 	if err != nil {
 		return nil, err
 	}
+	pdCli, err := pd.NewClientWithContext(ctx, []string{cfg.TiDB.PdAddr}, tls.ToPDSecurityOption())
```

Comment:
not closed?

Reply:
other places too, seems not close anywhere

Reply:
already addressed in https://github.com/pingcap/tidb/pull/45680/commits/f163612ceb7e65424ac66953d7fd33c92ae3c29e
PTAL again